pkgs/applications/editors/neovim/default.nix

```diff
@@ -65,6 +73,18 @@ in
       cmake
       gettext
       pkgconfig
+    ]
+    ++ optionals devMode [
+      include-what-you-use  # for scripts/check-includes.py
+      jq                    # jq for scripts/vim-patch.sh -r
+      doxygen
+    ];
+
+    # extra programs test via `make functionaltest`
+    checkInputs = [
```

Comment:
This isn't conditional on `devMode`. Is that intentional?

Reply:
yes: one can run the functionaltests without the tools necessary to develop on neovim.